### DISCUSSION TITLE
Module issues on Orion

### DISCUSSION THREAD

**Discussion by wzzheng90** [NOAA-EMC/global-workflow#2744] [General]
Hi,

I got a problem with the g-w after the recent upgrades on Orion.   When I type "source workflow/gw_setup.sh" or compile the UFS,
the error shows  as follows:

================
Lmod has detected the following error:  Cannot load module "stack-intel/2022.0.2" without these module(s) loaded:
   intel/2022.1.2

While processing the following module(s):
    Module fullname       Module Filename
    ---------------       ---------------
    stack-intel/2022.0.2  /work/noaa/epic/role-epic/spack-stack/orion/spack-stack-1.6.0/envs/gsi-addon-env/install/modulefiles/Core/stack-intel/2022.0.2.lua
    module_gwsetup.orion  /work/noaa/fv3-cam/wzheng/HR3/global-workflow/modulefiles/module_gwsetup.orion.lua

=========================
I do not find where to load intel/2022.1.2 module either.

Thank you!

Weizhong

**Comment by JessicaMeixner-NOAA** [comment]
@wzzheng90 See https://github.com/NOAA-EMC/global-workflow/issues/2694 and https://github.com/NOAA-EMC/global-workflow/pull/2741 to check the progress being made towards being able to run on orion again.

**Reply by wzzheng90** [reply]
@JessicaMeixner-NOAA   Thanks!!

**Comment by aerorahul** [comment]
I think this has been resolved.
https://github.com/NOAA-EMC/global-workflow/pull/2877 is the final PR bringing Orion back to being fully usable with cycling.
